<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test Learn NeuroNet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt_learn_neuro_net">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Learn NN</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_Lat">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_latitude">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>10</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_Lon">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>50</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_longitude">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>50</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title_accelerometer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accelerometer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_x_accelerometer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>150</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_x_accelerometer">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>150</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_y_accelerometer">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>180</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_y_accelerometer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>180</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_z_accelerometer">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>210</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_z_accelerometer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>210</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title_accelerometer_2">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>250</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gyroscope</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_y_gyroscope">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>330</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_z_gyroscope">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>360</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_x_gyroscope">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>300</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_z_gyroscope">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>360</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_y_gyroscope">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>330</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_x_gyroscope">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>300</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_calibrate_sensors">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CalibrateNull</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_send_read_command">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OnOff Relay</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_slave_14">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slave 0x14:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_slave_14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NaN</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bluetooth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_btserver_on_off">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>off</string>
    </property>
   </widget>
   <widget class="CustomLabel" name="lb_draw_track">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>90</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>240</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_current_epoha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epoha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_current_epoha">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_current_count">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_current_count">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_current_train">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_current_train">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train:</string>
    </property>
   </widget>
   <widget class="CustomLabel" name="lb_draw_error">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>1000</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CustomLabel" name="lb_draw_neuro">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>690</y>
      <width>1000</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_error_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Error graph</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nnet_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>211</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>NNet graph</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_current_error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_current_error">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>590</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>20</width>
      <height>360</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>20</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_max_mut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>log10 mutation:</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_max_mut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_max_mut">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_probality_mutation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_probality_mutation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_probality_mutation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>probality mutation:</string>
    </property>
   </widget>
   <widget class="QSlider" name="hs_generation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>-5</number>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lb_generation">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name_generation">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>271</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>log10 generation:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_calibrate_gyro_axisZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CalibrateZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>reset data</string>
    </property>
   </widget>
   <zorder>lb_draw_track</zorder>
   <zorder>lb_draw_error</zorder>
   <zorder>lb_draw_neuro</zorder>
   <zorder>lb_name_Lat</zorder>
   <zorder>lb_latitude</zorder>
   <zorder>lb_name_Lon</zorder>
   <zorder>lb_longitude</zorder>
   <zorder>lb_title_accelerometer</zorder>
   <zorder>lb_name_x_accelerometer</zorder>
   <zorder>lb_x_accelerometer</zorder>
   <zorder>lb_y_accelerometer</zorder>
   <zorder>lb_name_y_accelerometer</zorder>
   <zorder>lb_z_accelerometer</zorder>
   <zorder>lb_name_z_accelerometer</zorder>
   <zorder>lb_title_accelerometer_2</zorder>
   <zorder>lb_y_gyroscope</zorder>
   <zorder>lb_name_z_gyroscope</zorder>
   <zorder>lb_x_gyroscope</zorder>
   <zorder>lb_z_gyroscope</zorder>
   <zorder>lb_name_y_gyroscope</zorder>
   <zorder>lb_name_x_gyroscope</zorder>
   <zorder>lb_name_slave_14</zorder>
   <zorder>lb_slave_14</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>label</zorder>
   <zorder>lb_btserver_on_off</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>lb_name_current_epoha</zorder>
   <zorder>lb_current_epoha</zorder>
   <zorder>lb_current_count</zorder>
   <zorder>lb_name_current_count</zorder>
   <zorder>lb_current_train</zorder>
   <zorder>lb_name_current_train</zorder>
   <zorder>lb_nnet_graph</zorder>
   <zorder>lb_error_graph</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>lb_name_current_error</zorder>
   <zorder>lb_current_error</zorder>
   <zorder>line_9</zorder>
   <zorder>line_10</zorder>
   <zorder>lb_name_max_mut</zorder>
   <zorder>lb_max_mut</zorder>
   <zorder>lb_probality_mutation</zorder>
   <zorder>lb_name_probality_mutation</zorder>
   <zorder>lb_generation</zorder>
   <zorder>lb_name_generation</zorder>
   <zorder>hs_probality_mutation</zorder>
   <zorder>hs_max_mut</zorder>
   <zorder>hs_generation</zorder>
   <zorder>bt_calibrate_sensors</zorder>
   <zorder>bt_send_read_command</zorder>
   <zorder>bt_learn_neuro_net</zorder>
   <zorder>bt_calibrate_gyro_axisZ</zorder>
   <zorder>bt_reset</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCurrent_Data">
    <property name="title">
     <string>Не нажимать</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuCurrent_Data"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Я же просил...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>customlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
